service/fck-nat.sh

```diff
@@ -19,6 +19,12 @@ if test -n "$eni_id"; then
         --device-index 1 \
         --network-interface-id "$eni_id"
 
+    echo "Disabling source/destination checks on $eni_id..."
+    aws ec2 modify-network-interface-attribute \
+        --region "$aws_region" \
+        --network-interface-id "$eni_id" \
+        --no-source-dest-check
+
```

Comment:
No need for this. The ENI can/should be created with source/destination checks enabled.

Reply:
Yeah, it shouldn't be required, however:
* You can't set this if you're creating instances from a launch template (I.e. high-availability mode with an autoscaling group)
* It _seems_ to occasionally be reset in my testing (lots and lots of terminated and restated instances)

so this was added as a triple check to make sure it's all configured correctly.

Reply:
You wouldn't create the HA ENI as part of a launch template though, it would need to be outside. Unless src/dest checks would need to be disabled on eth0 as well which then I'd need to get the ENI ID here to do that.

Reply:
Sorry, I'm half in Terraform mode.

I _swear_ I saw the HA ENI get switched back after I changed it, but you're probably right and this can go away - I'm pretty sure that it's only needed on the HA ENI and those can be created with these checks disabled, but my fork of the Terraform module has belt-and-braces brute force solutions to the problems I found, hence this change here.

I'll drop this, one sec...